app/uk/gov/hmrc/customs/declarations/metrics/repo/MetricsRepo.scala

```diff
@@ -48,17 +48,26 @@ class MetricsMongoRepo @Inject() (mongoDbProvider: MongoDbProvider,
   private implicit val format: OFormat[ConversationMetrics] = ConversationMetrics.conversationMetricsJF
   private implicit val formatEvent: Format[Event] = Event.EventJF
 
+  private val ttlIndexName = "createdDate-Index"
+  private val ttlInSeconds = metricsConfig.ttlInSeconds
+  private val ttlIndex = Index(
+    key = Seq("createdAt" -> IndexType.Descending),
+    name = Some(ttlIndexName),
+    unique = false,
+    options = BSONDocument("expireAfterSeconds" -> ttlInSeconds)
+  )
+
+  dropInvalidIndexes.flatMap { _ =>
+    collection.indexesManager.ensure(ttlIndex)
+  }
+
   override def indexes: Seq[Index] = Seq(
     Index(
```

Comment:
Why not have both indexes are as private vals? Rather than just ttlIndex

Reply:
I'm just following the pattern of the other services (keep consistency). I did modify the way the indexes are arranged when I was upgrading the reactivemongo lib but that work had to be rolled back due to a bug platOps found in the new lib.